core/lib/core.ts

```diff
@@ -811,7 +844,7 @@ export function trackerCore(base64: boolean, callback?: (PayloadData) => void) {
 			tstamp?: Timestamp): PayloadData {
 
 			return trackSelfDescribingEvent({
-				schema: 'iglu:com.snowplowanalytics.snowplow/change_form/jsonschema/1-0-0',
+				schema: 'iglu:com.snowplowanalytics.snowplow/' + schema + '/jsonschema/1-0-0',
```

Comment:
now that we have a schema argument, is it reasonable to have a hard-coded version?

Reply:
Is something like this what you're talking about?
```
let event_schema = '';
if (schema === 'change_form'){
	event_schema = 'iglu:com.snowplowanalytics.snowplow/change_form/jsonschema/1-0-0';
} else if (schema === 'focus_form') {
    event_schema = 'iglu:com.snowplowanalytics.snowplow/focus_form/jsonschema/1-0-0';
}
```